src/js/components/Select/__tests__/Select-test.js

```diff
@@ -14,6 +14,7 @@ import { Select } from '..';
 
 describe('Select', () => {
   window.scrollTo = jest.fn();
+  window.HTMLElement.prototype.scrollIntoView = jest.fn();
```

Comment:
I've relocated this here because the tests I've introduced are also affected by this function, which doesn't appear to exist in JSDOM.

I noticed the previous location made no attempt at restoring the original contents, though I suspect it's an inconsequential matter.